.changeset/thin-buses-greet.md

```diff
@@ -0,0 +1,5 @@
+---
+"stylelint": patch
+---
+
+Fixed: `selector-pseudo-class-no-unknown` false positives
```

Comment:
https://github.com/stylelint/stylelint/blob/main/.changeset/dirty-fishes-approve.md will be removed